src/bentoml/_internal/configuration/__init__.py

```diff
@@ -14,11 +15,7 @@
     from ..._version import __version__
     from ..._version import __version_tuple__
 except ImportError:
-    # Since we aren't VCS _version.py, which is generated by setuptools_scm
-    # BentoML uses setuptools_scm 'post-release' for version scheme
-    from setuptools_scm import get_version
-
-    __version__ = get_version(version_scheme="post-release")
+    __version__ = importlib.metadata.version("bentoml")
     __version_tuple__ = (0, 0, 0, "dirty")
```

Comment:
Is `__version_tuple__` still needed? since we are not using setuptools-scm, `_version.py` won't be created.
`pdm-backend` also supports writing to version file, but `__version_tuple__` is not supported

Reply:
I was using hatch-vcs for this.

Reply:
We are still using `__version_tuple__` to handle the check if bentoml is installed with PyPI, in L87

Reply:
Oh right I thought you are using pdm-backend